modules/claims_api/app/workers/claims_api/ews_vbms_upload_job.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'claims_api/vbms_uploader'
+require 'claims_api/ews_vbms_sidekiq'
+
+module ClaimsApi
+  class EwsVBMSUploadJob
+    include Sidekiq::Worker
+    include ClaimsApi::EwsVBMSSidekiq
+
+    # Uploads an evidence waiver submission for a given claim to VBMS.
+    # If successfully uploaded, it queues a job to update the EWS code in BGS, as well.
+    #
+    # @param evidence_waiver_id [String] Unique identifier of the submitted EWS
+    def perform(evidence_waiver_id)
+      evidence_waiver = ClaimsApi::EvidenceWaiverSubmission.find(evidence_waiver_id)
+      uploader = ClaimsApi::EvidenceWaiverSubmissionUploader.new(evidence_waiver_id)
+      output_path = ClaimsApi::EvidenceWaiverBuilderJob.new.perform(evidence_waiver_id)
+      uploader.retrieve_from_store!(output_path)
+      file_path = fetch_file_path(uploader)
+      upload_to_vbms(evidence_waiver, file_path)
+
+      ClaimsApi::EwsUpdater.new.perform(evidence_waiver_id)
+
+      res = { details: { output_path: output_path, status: evidence_waiver.status } }
+      return res unless res.nil?
```

Comment:
we wouldn't need this. we're running the initial job from the controller as a `perform_async` so we wouldn't get a response from this anywho.